### PR TITLE
added selenium support, initAndIngestTest and MyDataPageTest

### PR DESCRIPTION
**Pull request by parthp2107**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>${selenium.version}</version>
+        </dependency>
+<!--        <dependency>-->
```
Comment (shahsank3t) [inline pom.xml]: Remove the unwanted code instead of commenting out if there's no reason for it.

File: selenium.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+path = "/Users/parthpanchal/Documents/CodeBase"
```
Comment (shahsank3t) [inline selenium.properties]: use a proper naming convention. is this the root path/directory of this selenium codebase or OpenMetadata codebase or what?

File: src/main/java/org/openmetadata/selenium/properties/Property.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        }
+        return instance;
+    }
+
+    private void loadData(){
+        Properties properties = new Properties();
+        try {
+            properties.load(new FileInputStream("selenium.properties"));
+        } catch (IOException ignored) {}
+        PATH = properties.getProperty("path");
```
Comment (shahsank3t) [inline src/main/java/org/openmetadata/selenium/properties/Property.java]: property names should be static/constant variables and not be used directly.

File: src/test/java/org/openmetadata/selenium/pages/MyDataPageTest.java [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    static WebDriver webDriver;
+    String homeXPath = "/html/body/div/div/div[2]/div[1]/div/div[1]/a";
+    Integer waitTime = Property.getInstance().getSleepTime();
+
+    @BeforeMethod
+    public void openMetadataWindow() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/drivers/macM1/chromedriver");
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+        webDriver.get("http://localhost:8585");
```
Comment (shahsank3t) [inline src/test/java/org/openmetadata/selenium/pages/MyDataPageTest.java]: This URL should come from the property file as one might decide to change the url:port anytime.

File: testng.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+    <suite name = "Suite1">
```
Comment (shahsank3t) [inline testng.xml]: Give proper name to suite

File: testng.xml [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+        <test name="Test1">
```
Comment (shahsank3t) [inline testng.xml]: Give proper name to test

**Review by shahsank3t — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (parthp2107) [reply pom.xml]: Addressed in latest commit

File: selenium.properties (hunk 2 above)
Comment (parthp2107) [reply selenium.properties]: Addressed in latest commit

File: src/main/java/org/openmetadata/selenium/properties/Property.java (hunk 3 above)
Comment (parthp2107) [reply src/main/java/org/openmetadata/selenium/properties/Property.java]: Addressed in latest commit

File: src/test/java/org/openmetadata/selenium/pages/MyDataPageTest.java (hunk 4 above)
Comment (parthp2107) [reply src/test/java/org/openmetadata/selenium/pages/MyDataPageTest.java]: Addressed in latest commit

File: testng.xml (hunk 5 above)
Comment (parthp2107) [reply testng.xml]: Addressed in latest commit

File: testng.xml (hunk 6 above)
Comment (parthp2107) [reply testng.xml]: Addressed in latest commit

**Review by shahsank3t — APPROVED** [review]

**Pull request merged**